### DISCUSSION TITLE
Will the components of PaddlePaddle be updated continuously

### DISCUSSION THREAD

**Discussion by crazycell** [deepjavalibrary/djl#1567] [Q&A]
I have some questions about DJLxPaddlePaddle:
1. Is the version of 'paddlepaddle-native-auto' equals  the version of PaddlePaddle?
2. If the answer to the above question is yes，will this component be updated continuously? Cause the latest verion of PaddlePaddle has been 2.2.2

**Comment by lanking520** [comment]
Yes. this is on our roadmap. Will follow up this thread with more updates

**Comment by lanking520** [comment]
https://github.com/deepjavalibrary/djl/pull/1601
